x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/SecurityTests.java

```diff
@@ -472,39 +472,6 @@ public void testIndexJoinValidator_FullyCurrentCluster() throws Exception {
         joinValidator.accept(node, clusterState);
     }
 
-    public void testIndexUpgradeValidatorWithUpToDateIndex() throws Exception {
```

Comment:
Shouldn't the above `testIndexJoinVAlidator_FullyCurrentCluster()` be removed as well?

Reply:
Good spot, removed